README.md

```diff
@@ -1 +1,95 @@
+
+![img](screenshot.png)
+
+
+## Built With
+
+- HTML5
+- CSS3
+- JavaScript
+- Styleint
+- Eslint
+
+## Live Demo
+
+[Live Demo Link](https://happy-jepsen-3ea201.netlify.app/)
+
+
+
+## Getting Started
+
+1. - Copy the git link `https://github.com/rloterh/MobileMenu.git` of this project
+2. - In your terminal run the command `git clone https://github.com/rloterh/MobileMenu.git`
+3. - Navigate to the cloned project directory
+4. - Within the project root directory launch `index.html` in a web browser
+
+
+## Testing
+
+This project was tested using [W3C Validator](https://validator.w3.org/), [Stylelint](https://stylelint.io/) and [Eslint](https://eslint.org/)
+ 
+
+### Validating HTML Syntax
+
+1. - Go to [The W3C Markup Validation Service](https://validator.w3.org/#validate_by_upload)
+2. - Upload the file named `index.html` from the project directory
+3. - Click on the check button to validate
+4. - All test cases should pass, when no error message is shown 
+
+### Validating CSS Syntax
+
+1. - Run the command `npm -v` to ensure node package manager is installed
+2. - Go to [npmjs.com](https://www.npmjs.com/get-npm) to install node package manager, if not installed
+3. - Go to the terminal, and navigate to project directory
+4. - Run the command `npm install`, if packages are not installed
+5. - Validate the css syntax and code standard by running the command `npx stylelint "**/*.{css,scss}"`
+6. - All test cases passes, when no error message is shown in the terminal
+
+### Validating JavaScrip Syntax
+
+1. - Run the command `npm -v` to ensure node package manager is installed
+2. - Go to [npmjs.com](https://www.npmjs.com/get-npm) to install node package manager, if not installed
+3. - Go to the terminal, and navigate to project directory
+4. - Run the command `npm install`, if packages are not installed
+5. - Validate the css syntax and code standard by running the command `npx eslint . `
+6. - All test cases passes, when no error message is shown in the terminal
+
+
+## Authors
+
+👤 **Robert Loterh**
+
+- GitHub: [@githubhandle](https://github.com/rloterh)
+- Twitter: [@twitterhandle](https://twitter.com/RLoterh)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/robert-loterh)
+
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/rloterh/MobileMenu/issues).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Application is based on this [Figma design](https://www.figma.com/file/t3EJUCAEViw3QasuJLPLVT/Microverse-Student-Potfolio-Templates-Main?node-id=1%3A1471)!
+- [Microverse](https://www.microverse.org/)!
+
+## 📝 License
+
+This project is [MIT](./MIT.md) licensed.
+
+
+
+
+
+
```

Comment:
To have professional documentation, I kindly invite you to delete this unnecessary blank space. This could help you to have a better readme file. 🐸